frodo.js

```diff
@@ -1,12 +1,12 @@
-/**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!